code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -414,12 +415,9 @@ components:
             - VM
             - CONTAINER
             - HELM 
-        fileName:
-          type: string
-          minLength:  8
-          maxLength: 32
-          description: Name of the file with the extension eg. zip, targz, etc.
-        repository:       
+        operatingSystem:
+          $ref: '#/components/schemas/OperatingSystem'
+        appRepo:       
```

Comment:
With the name "appRepo", this makes it sound like the URL below is the URL for the repository, i.e. docker.io, rather than the URL for the image, i.e. docker.io/hashicorp/httpecho:2.3.1. I suggest to rename the "url" field below to "imagePath", so it is clear the URL is for the image.

Reply:
I think this is valid. Will provide a change for this.

Reply:
Changes done and committed to PR